### PR TITLE
feat: Save Asset View Options Settings (#147)

### PR DESCRIPTION
**Pull request by dloe**
# PR Details

Saving out asset view options (Filter tags, zoom, etc) so that new instances of the editor dont repeatively use default values. Some of the view options actually are already able to be saved between instances, added the filter tags and tile and grid zoom to those saved out.

## Related Issue

Issue: https://github.com/stride3d/stride/issues/147#issuecomment-2157012923

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [X] **I have built and run the editor to try this change out.**

### CODE REVIEW

**Comment by Eideren** [comment]
Sorry about the delay, sounds like @Kryptos-FR is busy, I'll check back in on or two weeks if he hasn't looked at it by then

File: sources/editor/Stride.Core.Assets.Editor/ViewModel/AssetCollectionViewModel.cs [hunk 1]
```diff
@@ -153,6 +154,18 @@ public override int GetHashCode()
...
+        public sealed class AssetFilterViewModelData
+        {
+            public string DisplayName = "";
+            public string Filter = "";
+            public bool IsActive = false;
+            public FilterCategory category = FilterCategory.AssetName;
+        }
+
+        // Storing out primitive data for filters to save between instances of editor
+        List<AssetFilterViewModelData> StoredListData = InternalSettings.ViewFilters.GetValue();
```
Comment (Eideren) [inline sources/editor/Stride.Core.Assets.Editor/ViewModel/AssetCollectionViewModel.cs:167]: This field needs its private access modifier specified explicitly :)

**Comment by Eideren** [comment]
Looks good otherwise, thanks :)

File: sources/editor/Stride.Core.Assets.Editor/ViewModel/AssetCollectionViewModel.cs (hunk 1 above)
Comment (dloe) [reply sources/editor/Stride.Core.Assets.Editor/ViewModel/AssetCollectionViewModel.cs]: Added, should be good to go now

**Comment by Eideren** [comment]
Thanks !

**Pull request merged**